keps/sig-node/2625-cpumanager-policies-thread-placement/README.md

```diff
@@ -268,11 +271,30 @@ The [implementation PR](https://github.com/kubernetes/kubernetes/pull/101432) wi
 #### Alpha to Beta Graduation
 - [X] Gather feedback from the consumer of the policy.
 - [X] No major bugs reported in the previous cycle.
+- [X] Use of this policy option to further configure the behavior of CPU manager. Another CPUManager policy option `distribute-cpus-across-numa` is being proposed in 1.23 release to distribute CPUs across NUMA nodes instead of packing them.
 
 #### Beta to G.A Graduation
 - [X] Allowing time for feedback (1 year).
 - [X] Risks have been addressed.
 
+### Graduation Criteria of Options
+
+In 1.23 release, as we are graduating this feature to Beta meaning `CPUManagerPolicyOptions` is enabled by default allowing the user to configure CPU Manager static policy with the option `full-pcpus-only`.
+NOTE: Even though the feature gate is enabled by default the user still has to explicitly use the Kubelet flag called `CPUManagerPolicyOptions` in the kubelet config or command line argument called `cpumanager-policy-options` along with a specific policy option to use this feature.
+- In addition to this, in order to not have all alpha-quality experimental options introduced in the future available by default, we are introducing an additional feature gate called `CPUManagerPolicyExperimentalOptions` that controls all the experimental options. The experimental options are hidden by default and only if the feature gate is enabled the user has the ability to use the experimental options. Based on the graduation criteria described below, a policy option can move from being hidden to being non-hidden. Once the feature is non-hidden the user would not need to use `CPUManagerPolicyExperimentalOptions` feature gate in order to use that option.
```

Comment:
i understand why or how this may be useful in the future, but am i mistaken that there is no option gated by this once full-pcpus-only goes beta?  if so, why have a new feature gate at all?  wouldn't we need a feature gate per specific option?

Reply:
what is not clear to me with the new feature gate `CPUManagerPolicyExperimentalOptions` is if it would ever proceed out of alpha?

Reply:
we didn't want to gate every option individually, but rather just put all alpha options behind one gate. maybe after the feature itself goes stable, we will gate beta options behind another gate but that's TBD.

Are you saying there will be no gated options by this flag, this release? If so then we would add it only when needed.

Reply:
Conversation continuing on slack here: https://kubernetes.slack.com/archives/C0BP8PW9G/p1631123745114500.